### PR TITLE
Document links in log output

### PR DESCRIPTION
**Pull request by ticky**
Proposed change for #412

### CODE REVIEW

File: pages/pipelines/links_and_images_in_log_output.md.erb [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
+
+  printf '\033]1339;'$LINK'\a\n'
+}
+```
+
+You can use it like so:
+
+```bash
+inline_link "https://buildkite.com/"
+```
```
Comment (sj26) [inline pages/pipelines/links_and_images_in_log_output.md.erb]: Maybe also worth mentioning that you can link artifacts using "artifact://..." urls?

**Comment by sj26** [comment]
This looks good!

File: pages/pipelines/links_and_images_in_log_output.md.erb (hunk 1 above)
Comment (ticky) [reply pages/pipelines/links_and_images_in_log_output.md.erb]: Done!

Comment (sj26) [reply pages/pipelines/links_and_images_in_log_output.md.erb]: Nice :+1:

**Comment by ticky** [comment]
Uh-oh, this broke the images in this document…

**Comment by ticky** [comment]
Looks like it was actually a deploy bug!

**Comment by sj26** [comment]
The deploy bug should be fixed now 👌

**Pull request merged**